src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -61,17 +62,20 @@ export class ContentTypesEditComponent implements OnInit, OnDestroy {
         private fieldService: FieldService,
         private route: ActivatedRoute,
         public dotMessageService: DotMessageService,
-        public router: Router
+        public router: Router,
+        private editContentTypeCacheService: EditContentTypeCacheService
     ) {}
 
     ngOnInit(): void {
+
         this.route.data
             .pipe(
                 pluck('contentType'),
                 takeUntil(this.destroy$)
             )
             .subscribe((contentType: ContentType) => {
                 this.data = contentType;
+                this.editContentTypeCacheService.contentType = contentType;
```

Comment:
We talked about this with the pagination, I think we said we should use methods to set things in services instead of just setting the property directly.